lib/service/token_list.dart

```diff
@@ -130,6 +130,15 @@ class TokenList {
       );
     }),
   ];
+
+  static String mapTickerToName(String ticker) {
+    try {
+      final name = tokenList.firstWhere((token) => token.ticker == ticker).name;
+      return name;
+    } catch (e) {
+      return '';
+    }
```

Comment:
This logic is questionable.
You should get rid of the `try catch` and utilize the `firstWhereOrNull` from package `collection`.
You could then do: `tokenList.firstWhereOrNull((token) => token.ticker == ticker)?.name ?? ''`